tools/read_cmt.ml

```diff
@@ -17,20 +17,26 @@ let gen_annot = ref false
 let gen_ml = ref false
 let print_info_arg = ref false
 let target_filename = ref None
+let save_cmt_info = ref false
 
-let arg_list = [
+let arg_list = Arg.align [
   "-o", Arg.String (fun s -> target_filename := Some s),
     " FILE (or -) : dump to file FILE (or stdout)";
   "-annot", Arg.Set gen_annot, " : generate the corresponding .annot file";
+  "-save-cmt-info", Arg.Set save_cmt_info,
+  " Encapsulate additional cmt information in annotations";
   "-src", Arg.Set gen_ml,
-    " : convert .cmt or .cmti back to source code (without comments)";
+    " Convert .cmt or .cmti back to source code (without comments)";
   "-info", Arg.Set print_info_arg, " : print information on the file";
   "-args", Arg.Expand Arg.read_arg,
     " <file> Read additional newline separated command line arguments \n\
     \      from <file>";
   "-args0", Arg.Expand Arg.read_arg0,
     "<file> Read additional NUL separated command line arguments from \n\
     \      <file>";
+  "-I", Arg.String (fun s ->
+    Clflags.include_dirs := s :: !Clflags.include_dirs
+  ), "DIR Directories for includes";
```

Comment:
Couldn't this be worded more like driver/main_args.ml (`<dir>  Add <dir> to the list of include directories`)